app/lib/status.py

```diff
@@ -13,11 +13,29 @@ def __init__(self, message, details=None, result=None, info=None):
 
     def toDict(self):
         return {
-            'result': self.result,
             'info': self.info,
             'status': {
                 'statusCode': self.statusCode,
                 'statusMsg': self.statusMsg,
-                'statusDetails': self.statusDetails,
+                'statusDetails': self.statusDetails
             },
+            'result': self.result
         }
+
+
+class BadRequest(TodoException):
+    """Exception where bad request is made.
```

Comment:
What do you think about changing this to `Exception raised when a bad client request is made.`? Codes in the 400 range are errors raised because the client messed up somehow.